pkg/config/config.go

```diff
@@ -711,6 +711,49 @@ func (c *Config) GetDefaultEnv() []string {
 	return append(env, c.Containers.Env...)
 }
 
+// Capabilities returns the capabilities parses the Add and Drop capability
+// list from the default capabiltiies for the container
+func (c *Config) Capabilities(user string, addCapabilities, dropCapabilities []string) []string {
+
+	userNotRoot := func(user string) bool {
+		if user == "" || user == "root" || user == "0" {
+			return false
+		}
+		return true
+	}
+
+	var capabilities []string
+	defaultCapabilities := c.Containers.DefaultCapabilities
+	if userNotRoot(user) {
+		defaultCapabilities = []string{}
+	}
+
+	mapCap := make(map[string]bool, len(defaultCapabilities))
+	for _, c := range addCapabilities {
+		if strings.ToLower(c) == "all" {
+			defaultCapabilities = caps.GetAllCapabilities()
```

Comment:
Can we add a big `// TODO - get rid of this dependency`? We're pulling in docker, containerd and all kinds of other deps. We may open an issue here as well to not forget about it. I haven't looked at this package yet but I assume it's fairly simple to get all caps.

Reply:
Looking at https://github.com/moby/moby/blob/master/oci/caps/utils.go#L13, we can follow what init does and get rid of the other dependencies. We may add a comment were we copied the function from.

Reply:
I pulled in caps.go into this repo so that we did not pull in containerd and friends.

common does not pull in docker or containerd.